docs/border-width.md

```diff
@@ -88,8 +90,19 @@ Use the `border-{x|y}-{width}` utilities to set the border width on two sides of
 <div class="border-y-4 ..."></div>
 ```
 
+### Arbitrary values
+If you need to use a one-off `border-{width}` value, use square brackets to generate a property using any arbitrary width.
+This also works with or without an optional direction.
+`border-{x|y|l|r|t|b}-[{any width}]`
```

Comment:
I'm not sure about the use of `any width` 😄  I see that in Tailwind docs for border classes with arbitrary values they mention `border-{side}-[{width}]` to explain the pattern: thttps://tailwindcss.com/docs/border-width#arbitrary-values
I think the context of `one-off` makes it clear that whatever is in square brackets is some/any width value. Also, I like the use of `side` instead of `{x|y|l|r|t|b}` in their docs. From examples in the table it should already be clear what `side` refers, to in my opinion, and it makes it easier to see what it is that actually changed in the given pattern. But that's just my view of it - what do you think? 😊

Reply:
I'll look in to and re-work this. 🙂 👍